components/typography/styles.js

```diff
@@ -407,3 +489,42 @@ export const fontFacesCss = `@font-face {
 		url(${new URL(`${fonts.BCSansBoldItalic}.woff2`, importUrl)}) format('woff2'),
 		url(${new URL(`${fonts.BCSansBoldItalic}.woff`, importUrl)}) format('woff');
 }`;
+
+export const baseTypographyStyles = css`
+	${unsafeCSS(fontFacesCss)}
+	html {
```

Comment:
These variables were only ever included in tests and demo pages, not the LMS itself. I think they should be.

I also switched this from `*` and `html[dir="rtl"] *` to only set the variables on the HTML element. This aligns with where we set our other CSS variables (colors, etc.) and feels better since variables cascade anyway. Am I missing something?